data.json

```diff
@@ -520,5 +520,11 @@
     "githubUsername": "webdevrachit",
     "imageUrl": "https://user-images.githubusercontent.com/72686780/106394124-fa89d480-6420-11eb-837b-fbf179a7697d.png",
     "issueNumber": 286
-  }
+  },
+  {
+    "name": "Ankush Chauhan",
+    "githubUsername": "An-prog-hub",
+    "imageUrl": "https://user-images.githubusercontent.com/67872399/106575972-a71f9f80-6562-11eb-87dc-1b97d93f4d71.png",
+    "issueNumber": 310
+  },
```

Comment:
```suggestion
  }
```

Reply:
As Allan said, the comma could be removed 🤓 (I think that should fix the prettier fails)